### DISCUSSION TITLE
ocr连续识别，出现识别结果互相混入的严重问题

### DISCUSSION THREAD

**Discussion by 1Edwar** [PaddlePaddle/PaddleOCR#13001] [Q&A]
目前paddleocr部署在docker微服务中，请求量为秒级几十张，有几率出现相邻请求的两张图片其中一张的结果中会混入另一张的结果中，求问题归因和解决方法

- 系统环境/System Environment：linux debian，GPU是T4
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：识别+检测都是v3 mobile

**Comment by iwanglei1** [comment]
+1

**Comment by WLJ0314** [comment]
+1

**Comment by ParkourLiu** [comment]
+1
